src/components/templates/Camp/components/MapRoomOverlay.tsx

```diff
@@ -97,23 +97,20 @@ export const MapRoomOverlay: React.FC<MapRoomOverlayProps> = ({
     (e) => {
       e.stopPropagation();
 
-      if (isExternalUrl(room.url)) {
-        openRoomUrl(room.url);
-      } else {
-        window.location.href = getRoomUrl(room.url);
-      }
-
```

Comment:
nit: commented code

Reply:
It leaves context to the speedy hack being done here. And all of this is 🔥  in https://github.com/sparkletown/sparkle/pull/840 refactor anyway.